Dockerfile

```diff
@@ -1,75 +1,29 @@
-# BSD License
```

Comment:
If the Dockerfile has any bits from the original code, then keep it (it says it: "Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.")
If not, there's no need for the "Modifications Copyright" either

Reply:
in my mac, i cannot build
after this issue cleared, i will approve

Reply:
I see, I have removed the statement now